services/src/main/resources/locale/portlet/news/News_en.properties

```diff
@@ -214,3 +214,4 @@ news.publishTargets.managementDrawer.btn.confirm=Confirm
 news.publishTargets.managementDrawer.btn.cancel=Cancel
 news.publishTargets.managementDrawer.placeholder.name=Enter the target name
 news.publishTargets.managementDrawer.placeholder.description=Enter a description here
+news.publishTargets.managementDrawer.sameTargetWarning=A target with the same name already exists.
```

Comment:
```suggestion
news.publishTargets.managementDrawer.sameNewsTargetWarning=A news target with the same name already exists.
```

Reply:
the message should be "A target with the same name already exists" like in US specification